Dalamud/Interface/Internal/Windows/Data/TaskSchedulerWidget.cs

```diff
@@ -0,0 +1,256 @@
+        }
+
+        ImGui.SameLine();
+
+        if (ImGui.Button("As long as it's in Framework Thread"))
+        {
+            Task.Run(async () => await Service<Framework>.Get().RunOnFrameworkThread(() => { Log.Information("Task dispatched from non-framework.update thread"); }));
+            Service<Framework>.Get().RunOnFrameworkThread(() => { Log.Information("Task dispatched from framework.update thread"); }).Wait();
+        }
+
+        if (ImGui.Button("Drown in tasks"))
+        {
+            var token = this.taskSchedulerCancelSource.Token;
+            Task.Run(
+                () => 
+                {
+                    for (var i = 0; i < 100; i++)
+                    {
+                        token.ThrowIfCancellationRequested();
+                        Task.Run(
+                            () => 
+                            {
+                                for (var j = 0; j < 100; j++)
+                                {
+                                    token.ThrowIfCancellationRequested();
+                                    Task.Run(
+                                        () => 
+                                        {
+                                            for (var k = 0; k < 100; k++)
+                                            {
+                                                token.ThrowIfCancellationRequested();
+                                                Task.Run(
+                                                    () => 
+                                                    {
+                                                        for (var l = 0; l < 100; l++)
+                                                        {
+                                                            token.ThrowIfCancellationRequested();
+                                                            Task.Run(
+                                                                async () => 
+                                                                {
+                                                                    for (var m = 0; m < 100; m++)
+                                                                    {
+                                                                        token.ThrowIfCancellationRequested();
+                                                                        await Task.Delay(1, token);
+                                                                    }
+                                                                });
+                                                        }
+                                                    });
+                                            }
+                                        });
+                                }
+                            });
+                    }
+                });
+        }
```

Comment:
I know this isn't your code. Just... *pls*.

Reply:
I don't know what you are asking me to address with this comment.

Reply:
Nothing, just flagging it in general. I'll resolve this but I do want to draw maintainer attention to the fact that this exists.